### PR TITLE
Add Shin Megami Tensei: Devil Survivor 2 Record Breaker

### PR DESCRIPTION
**Pull request by robzombie91**
Adds Shin Megami Tensei: Devil Survivor 2 Record Breaker.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/shin-megami-tensei-devil-survivor-2-record-breaker/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Shin Megami Tensei: Devil Survivor 2 Record Breaker"
+description = "Your days are numbered. This is a chilling reminder that the apocalypse is upon us. Exclusive to the Nintendo 3DS handheld system, Shin Megami Tensei: Devil Survivor 2 Record Breaker offers more than 100 hours of gameplay and plot -- nearly double the content of the original Devil Survivor 2. It also includes fully voiced English story dialogue, giving new depth to the cast of Devil Survivor 2's original story."
```
Comment (Leo626) [inline games/shin-megami-tensei-devil-survivor-2-record-breaker/game.dat]: The description should be taken from Wikipedia (genre, developer, publisher, etc) but without the release dates.

**Comment by Leo626** [comment]
* Screenshots should not have the title and status bars. Use ShareX to capture the region within window and not the window itself.
* Three screenshots minimum if the game can progress pass the title screen.
* The wiki should be written objectively. Nothing personal like "I". The NG-Bad word list part should be omitted since users are expected to have their system archives dumped.

**Comment by robzombie91** [comment]
https://i.imgur.com/IxzzKiY.png - screenshot 1 title menu
https://i.imgur.com/BPmGcT7.png - screenshot 2 in-game
https://i.imgur.com/PRfyVua.png - screenshot 3 event-menu

**Comment by robzombie91** [comment]
Maybe a better Description: The setting of Devil Survivor 2 spans the whole country of Japan. It tells the story of Japanese high school students who enter a pact, enabling them to summon demons and fight mysterious creatures invading and destroying the country. The plot may unfold differently depending on the player's actions who assumes the role of a silent young demon tamer involved in this incident. The game is a tactical role-playing which takes after its predecessor as it involves controlling several demon tamers in their fights against different enemies and gaining new skills in the process. It has new additions such as the Fate System where the player can relate with other cast as well as the upgradeable racial skills.

**Comment by Leo626** [comment]
Your screenshots should be uploaded to your own repository [here](https://github.com/robzombie91/citra-games-wiki/tree/pr-builder-shin-megami-tensei-devil-survivor-2-record-breaker-1/games/shin-megami-tensei-devil-survivor-2-record-breaker/screenshots). Delete the one there first.

The description should be like wikipedia's and be relatively short. So something like this:

 > Shin Megami Tensei: Devil Survivor 2 Record Breaker is a role-playing game and enhanced port of Shin Megami Tensei: Devil Survivor 2. It was published by Atlus in North America and NIS America in Europe.

**Pull request merged**